assets/queries/cloudFormation/aws/s3_bucket_with_all_permissions/metadata.json

```diff
@@ -3,7 +3,7 @@
   "queryName": "S3 Bucket With All Permissions",
   "severity": "HIGH",
   "category": "Access Control",
-  "descriptionText": "S3 Buckets must not have all permissions, as to prevent leaking private information to the entire internet or allow unauthorized data tampering / deletion. This means the 'Effect' must not be 'Allow' when the 'Action' is '*', for all Principals.",
+  "descriptionText": "S3 Buckets should not have all permissions, as to prevent leaking private information to the entire internet or allow unauthorized data tampering / deletion. This means the 'Effect' must not be 'Allow' when the 'Action' is '*', for all Principals.",
```

Comment:
The query implementation approach for CF is different. IMHO, we should use a unique approach